runtime/queries/protobuf/highlights.scm

```diff
@@ -23,7 +24,6 @@
 
 [
   (mapName)
-  (oneofName)
```

Comment:
Why is this removed?

Reply:
I forgot to remove it apparently. One ofs aren't types, they are represented as fields so removing it makes the highlighting make more sense.